runtime/src/main/java/org/evosuite/runtime/vnet/NonFunctionalRequirementExtension.java

```diff
@@ -0,0 +1,25 @@
+package org.evosuite.runtime.vnet;
```

Comment:
Just a quick suggestion: I think it makes sense if we also add the copyright header to new files :slightly_smiling_face:

Reply:
Running `mvn license:format` adds license headers to all files that don't have one, if I remember correctly

Reply:
I just ran the command and a git diff showed that ~160 files dont have a license header. Shall I wait for this and the other pull request being merged to generate the license headers and update/generate them in a different pull request. Otherwise the pull requests may become messy?

Reply:
OK, I agree, a separate pull request for the license headers alone seems more appropriate then